pkg/requestlog/route.go

```diff
@@ -15,16 +15,22 @@ type Route struct {
 	TrackHTTPResponseTime func(string) func(int)
 	MetricsEndpoint       string
 
+	TrackHTTPMetrics func(*Route) func(int)
+
 	LogContent bool
 }
 
 var _ http.Handler = (*Route)(nil)
 
 func (rt *Route) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	crw := w.(*CountingResponseWriter)
+	// TODO: replace TrackHTTPResponseTime w/TrackHTTPMetrics
```

Comment:
Should we add a deprecation warning on top of `TrackHTTPResponseTime`?

https://github.com/golang/go/wiki/Deprecated